src/tv2_afvd_showstyle/migrations/index.ts

```diff
@@ -265,6 +267,12 @@ export const showStyleMigrations: MigrationStepShowStyle[] = [
 	renameTableColumn('1.7.9', 'OverlayShowMapping', 'GraphicsSetup', 'GfxSetup'),
 	renameTableId('1.7.9', 'OverlayShowMapping', 'GfxShowMapping'),
 
+	/**
+	 * 1.8.0 Move SelectedGfxSetupName to GFX Defaults
+	 */
+	moveSelectedGfxSetupNameToGfxDefaults('1.8.0', 'SelectedGfxSetupName', 'GfxDefaults'),
+	moveSelectedGfxSetupNameToGfxDefaultsInVariants('1.8.0', 'SelectedGfxSetupName', 'GfxDefaults'),
```

Comment:
Staging is already at 1.8.1 (and it was already deployed an migrations ran), so you'll probably need to bump these versions here (same in package.json) for a smooth deployment

Reply:
Oh, and are those supposed to only run for Galleries, or do we need to migrate Qboxes too? I'm not sure

Reply:
Very good point, and i believe they should run for QBox, good catches! I'll have it in the next commit!